mobile/app/(root)/(screens)/chatbot.tsx

```
@@ -1,152 +1,296 @@
+import { Feather } from "@expo/vector-icons";
+import FontAwesome from "@expo/vector-icons/FontAwesome";
+import FontAwesome5 from '@expo/vector-icons/FontAwesome5';
+
+export default function Chat() {
+  const backendUrl = `${ip}:3007/query`; 
+
+  const [chatText, setChatText] = useState("");
+  const [textInputHeight, setTextInputHeight] = useState(60);
+  const [sendingChat, setSendingChat] = useState(false);
+  const [conversation, setConversation] = useState<
+    { role: String; content: string }[]
```

Biome (1.9.4)

[error] Don't use 'String' as a type.

Use lowercase primitives for consistency.
Safe fix: Use 'string' instead


(lint/complexity/noBannedTypes)